src/index.js

```diff
@@ -15,14 +15,16 @@ const REGEXP_CONTENTHASH = /\[contenthash(?::(\d+))?\]/i;
 const REGEXP_NAME = /\[name\]/i;
 
 class CssDependency extends webpack.Dependency {
-  constructor({ identifier, content, media, sourceMap }, context, identifierIndex) {
+  constructor({ identifier, content, media, sourceMap }, module, identifierIndex) {
     super();
     this.identifier = identifier;
     this.identifierIndex = identifierIndex;
     this.content = content;
     this.media = media;
     this.sourceMap = sourceMap;
-    this.context = context;
+    this.context = module.context;
+    this.request = module.request;
```

Comment:
This is incorrect. `module` point point to multiple css modules i. e. when using `@import`. Using the `request` from `module` is wrong in these cases.